sdk/python/feast/client.py

```diff
@@ -494,7 +492,7 @@ def get_batch_features(
         self,
         feature_refs: List[str],
         entity_rows: Union[pd.DataFrame, str],
-        project: str = None,
+        default_project: str = None,
```

Comment:
Why was this change made?

Reply:
A merge conflict started causing an error in test cases. To fix that, I did this change, But i missed checking which one is latest. I will revert this change and also update test cases.